### PR TITLE
Updated Topics enhancement post subhead

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

File: site/en/blog/topics-enhancements/index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 layout: 'layouts/blog-post.njk'
 title: Enhancements to the Topics API
 description: >
-  Updates to the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
+ Updates to top topics selection, the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
 subhead: >
-  Updates to the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
+  Updates to top topics selection, the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
```
Comment (nancymic2) [inline site/en/blog/topics-enhancements/index.md:7]: comma here? or no? " ...selection, the Topics taxonomy, ..."

**Review by nancymic2 — APPROVED** [review]

File: site/en/blog/topics-enhancements/index.md (hunk 1 above)
Comment (samdutton) [reply site/en/blog/topics-enhancements/index.md:7]: I think it's OK as-is, because the sense is "the Topics ... filtering mechanisms".

...but I know it's getting a bit wordy :).
